lib/codegen/fromJsonSchema/cto/jsonSchemaVisitor.js

```diff
@@ -614,7 +638,10 @@ class JsonSchemaVisitor {
         };
 
         // Handle reserved properties.
-        if (['$identifier', '$class', '$timestamp'].includes(propertyName)) {
+        if (privateReservedProperties.includes(propertyName)) {
```

Comment:
Thanks for your contribution @kailash360 :). We already have a utility [function ](https://github.com/accordproject/concerto/blob/51dc8eb141ebb142b6964a25830fdeb21e147909/packages/concerto-core/lib/modelutil.js#L286) for the check.
Please use this.

Reply:
Done, I have replaced them with the ModelUtil methods.